src/App.js

```diff
@@ -15,6 +16,8 @@ function App() {
         component={SignIn}/>
         <Route path="/signup"
         component={SignUp}/>
+        <Route path="/home"
```

Comment:
The route to the home page should simply be `/` and not `/home`

Reply:
Sir i put it so because the first page a user will see is the dashboard where he will fine the link to the login or sign up form
after submitting the form now it will go the home page